src/directives/bind.js

```diff
@@ -68,15 +68,15 @@ export function handleAttributeBindingDirective(component, el, attrName, express
             const newClasses = value.split(' ').filter(Boolean)
             el.setAttribute('class', arrayUnique(originalClasses.concat(newClasses)).join(' '))
         }
-    } else if (isBooleanAttr(attrName)) {
-        // Boolean attributes have to be explicitly added and removed, not just set.
-        if (!! value) {
-            el.setAttribute(attrName, '')
+    } else {
+        // If an attribute's bound value is "null", remove the attribute 
+        // If a Boolean attribute has a falsy value, remove the attribute
+        const shouldRemoveAttr = value === null || (isBooleanAttr(attrName) && !value)
```

Comment:
Can we not just move this inside the statement instead of assigning to a const, it's only being used once and the comment above should suffice.

Reply:
Waiting on whether or not we want to check undefined/false.

But yeah I don't mind inlining it